recipes/spyder-line-profiler/run_test.py

```diff
@@ -0,0 +1,9 @@
+"""
+Test whether spyder_line_profiler is installed
+
+The test is only whether the module can be found. It does not attempt
+to import the module because this needs an X server.
+"""
+
+import imp
+imp.find_module('spyder_line_profiler')
```

Comment:
The import fails because it requires a connection to the X server on Unix which is not available on your CI servers. Therefore, the test is only whether the module can be found but not whether it can be imported.